manifests/rhoai/kubeflow-training-roles.yaml

```diff
@@ -0,0 +1,107 @@
+#This file has been copied from  ../overlays/kubeflow
+#The original labels have ben commented out for documentation purposes
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I don't think that ClusterRole is needed.

Reply:
Ok, removing it for now.